crates/net/network/src/transactions/mod.rs

```diff
@@ -88,14 +88,18 @@ use self::constants::{tx_manager::*, DEFAULT_SOFT_LIMIT_BYTE_SIZE_TRANSACTIONS_B
 /// The future for inserting a function into the pool
 pub type PoolImportFuture = Pin<Box<dyn Future<Output = Vec<PoolResult<TxHash>>> + Send + 'static>>;
 
-/// Api to interact with [`TransactionsManager`] task.
+/// Api to interact with [`TransactionsManager`] task. Only used in testnet type
```

Comment:
while we only used it for testing, this can be obtained via:

https://github.com/paradigmxyz/reth/blob/9f91c6ad94f94b2ff1a988c932c9ef0772c5bcc1/crates/net/network/src/transactions/mod.rs#L297-L300

there's also: 

https://github.com/paradigmxyz/reth/issues/5598

which aims to make this obtainable via a command on NetworkHandle